README.md

```diff
@@ -14,6 +14,7 @@ For a history on how this simulator started and the challenges on its way visit
 - CMake
 - SDL2 (provides the simulator window, handles mouse and keyboard input)
 - Compiler (g++ or clang++)
+- Node.js v12.0.0 or later for font generation with InfiniTime v1.10
```

Comment:
please move this note as sub-point to the line below, as the node.js part is a requirement of lv_font_conv. I intentionally left it out as is done in InfiniTime as well. The setup requirements are documented in the lv_font_conv project readme. But if it is a common pitfall we can mention it in the readme.

Something like "Note: requires Node.js v12.0.0 or later"

we can leave out the "since InfiniTime v1.10" part as for lv-conv the InfiniTime PR is linked

Reply:
Sounds good. These changes make it more clear.